css/main.css

```diff
@@ -0,0 +1,356 @@
+  background-color: white;
+}
+
+.logo {
+  width: 80px;
+  height: 80px;
+  padding: 10px;
+}
+
+.logo > h1 {
+  color: #6070ff;
+  font-size: 18px;
+  font-weight: bold;
+}
+
+header > .button {
+  padding: 15px;
+  color: #6070ff;
+}
+
+.section1 {
+  height: 550px;
+  background-color: white;
+  border-bottom-left-radius: 7rem;
+  padding-left: 25px;
+  padding-right: 25px;
+}
+
+.hero-section {
+  background-image: url("../image/heroImage.jpg");
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+.title {
+  text-align: left;
+  font-weight: bold;
+}
+
+.title > h1 {
+  font-size: 40px;
+  margin-top: 12px;
+  margin-bottom: 0;
+  color: #172b4d;
+}
+
+.description {
+  margin: auto;
+  font-size: 16px;
+  text-align: left;
+  font-weight: regular;
+  color: #344563;
+}
+
+ul {
+  display: flex;
+  flex-direction: row;
+  list-style-type: none;
+  justify-content: flex-start;
+  margin: 0;
+  padding: 0;
+}
+
+.icons > ul > li > a {
+  margin-left: 20px;
+  margin-top: 5px;
+  color: #505f79;
+  height: 20px;
+  width: 20px;
+}
+
+.icons > ul > li > a:hover { color: #7f8cff; }
+
+.social {
+  color: #7f8cff;
+  font-weight: medium;
+  font-size: 16px;
+}
+
+.section2 {
+  margin-top: 30px;
+  text-align: center;
+  margin-left: 20px;
+  margin-right: 20px;
+}
+
+.my-work {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  border-radius: 8px;
+  flex-wrap: wrap;
+  background-color: white;
+  padding: 10px;
+  margin-top: 20px;
+}
+
+.screenshot {
+  width: 100%;
+  text-align: center;
+  border-radius: 8px;
+}
+
+.screenshot > img {
+  width: 100%;
+  height: 100%;
+  border-radius: 10px;
+  display: flex;
+}
+
+.my-work > .title > h1 {
+  font-size: 32px;
+  font-weight: bold;
+  color: #172b4d;
+}
+
+.sub-headings > ul {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+}
+
+.sub-headings > ul > .bold {
+  font-size: 13px;
+  color: #344563;
+  text-transform: uppercase;
+  marker: none;
+}
+
+.sub-headings > ul > .marked {
+  font-size: 13px;
+  color: #7a869a;
+  list-style-type: disc;
+}
+
+.section2 > .description {
+  font-size: 15px;
+  font-weight: regular;
+}
+
+.tags {
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+  justify-content: flex-start;
+}
+
+.tags > ul > li {
+  color: #6070ff;
+  margin-left: 10px;
+  background-color: #ebebff;
+  padding-left: 10px;
+  padding-right: 10px;
+  border-radius: 5px;
+}
+
+.seeproject > button {
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-left: 20px;
+  padding-right: 20px;
+  border-color: #6070ff;
+  margin-top: 20px;
+  background-color: white;
+  border-radius: 8px;
+  display: flex;
+}
+
+.get-resume > button {
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-left: 20px;
+  padding-right: 20px;
+  border-color: #6070ff;
+  margin-top: 20px;
+  background-color: white;
+  border-radius: 8px;
+  display: flex;
+}
+
```

Comment:
Please kindly implement the active, pressed and hover effects of all the buttons. It is true that these effects are not seen on mobile versions but it is a must to implement them at this stage of the project :stuck_out_tongue_winking_eye:  You can check the desktop side of the figma design for the button effects. :pray: